linalg/densemat.cpp

```diff
@@ -528,7 +528,7 @@ double DenseMatrix::Weight() const
       double F = d[0] * d[3] + d[1] * d[4] + d[2] * d[5];
       return sqrt(E * G - F * F);
    }
-   mfem_error("DenseMatrix::Weight()");
+   mfem_error("DenseMatrix::Weight(): dimension mismatch");
```

Comment:
"dimension mismatch" is not the best description of this error.  Perhaps something like "unsupported dimensions".

Reply:
I agree, but the first conditional here is
```
if (Height() == Width())
```
then there are some specializations for smaller dimensions where height and width are different so maybe "mismatched or unsupported dimensions" would be better.